modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -133,4 +133,4 @@ platform:
 ----
 <1> Replace `<bootstrap_os_image>` with the value of `$BOOTSTRAP_OS_IMAGE`.
 +
-See the "Configuring the install-config.yaml file" section for additional details.
+See the "Configuring the install-config.yaml file" section for additional details.
```

Comment:
This line makes no sense. Suggest removing it. It is referring to the next section in the doc. What is the point of having it here at all?